apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,109 @@
+
+In addition to providing the initial state of a simulation, structured data can be used directly for verification and validation purposes, including as a means of backtesting simulation experiments, and individual behaviors themselves. In a world where behaviors and whole simulation models may be generated by foundation models, this provides humans with an improved sense of their likely correctness.
+
+## More for humans than machines
+
+All this serves to highlight an interesting possibility. If to date structured data has existed for the benefit of _machines_, is it possible that in the future its primary use will in fact me to us, _humans_, in verifying and validating the outputs of machine-generated models: simulation, language, multimodal, or other?
+
+Inherently tied to the idea of structured data are _types_. Data may be of a certain 'type' if it follows a set of rules established by that type.
+
+One way of thinking about structured data is the [Block Protocol type system](https://blockprotocol.org/docs/types), which outlines three different kinds of types: data types, property types, and entity types.
+
+1. **Entites** (semantic 'things' in a database) have entity types.
+  - **Entity types** describe the _properties_ and type(s) of any outgoing _link entities_ that can be associated with an entity.
+  - **Link entities** are the relationships, or 'things' that connect other entities. For example, one `Person` entity may be connected by a linked entity called `Spouse` to another `Person`.
+  - Right now, entities can only have one entity type, but we intend to support entities with multiple entity types in the specification soon.
+2. **Properties** contain data associated with some particular aspect of an entity.
+  - **Property types** describe the possible values can be associated with a property, by way of reference to _data types_ and other properties.
+3. **Data types** describe a space of possible valid values (e.g. `string`, `number` or `boolean`).
+  - Support for defining "custom" data types, i.e. "non-primitive" data types which are not found in the Block Protocol's [graph module specification](https://blockprotocol.org/docs/spec/graph) is coming soon.
+
+The open-source [HASH type editor](https://github.com/hashintel/hash/tree/main/libs/%40hashintel/type-editor) makes it easy to create and edit Block Protocol types.
+
+Both defining types in the first place, and the act of typing data, minimize the potential for errors, improve the readability or intelligibility of data, and can enable optimized processing of it. A piece of information's type can be used to ascertain specific meaning and purpose, as well as confirm its validity.
```

Comment:
I initially thought 'Both...' was still referring to the the type editor's capabilities. I think removing it avoids that potential for confusion.
```suggestion
Defining types in the first place, and the act of typing data, minimize the potential for errors, improve the readability or intelligibility of data, and can enable optimized processing of it. A piece of information's type can be used to ascertain specific meaning and purpose, as well as confirm its validity.
```